projects/aca-shared/src/lib/components/open-in-app/open-in-app.component.ts

```diff
@@ -0,0 +1,54 @@
+/*!
+ * @license
+ * Alfresco Example Content Application
+ *
+ * Copyright (C) 2005 - 2020 Alfresco Software Limited
+ *
+ * This file is part of the Alfresco Example Content Application.
+ * If the software was purchased under a paid Alfresco license, the terms of
+ * the paid license agreement will prevail.  Otherwise, the software is
+ * provided under the following open source license terms:
+ *
+ * The Alfresco Example Content Application is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Lesser General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * The Alfresco Example Content Application is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with Alfresco. If not, see <http://www.gnu.org/licenses/>.
+ */
+
+import { Component, Inject, OnInit } from '@angular/core';
+import { MatDialogRef, MAT_DIALOG_DATA } from '@angular/material/dialog';
+
+@Component({
+  selector: 'aca-open-in-app',
+  templateUrl: './open-in-app.component.html',
+  styleUrls: ['./open-in-app.component.scss']
+})
+export class OpenInAppComponent implements OnInit {
+  private redirectUrl: string;
+  constructor(
+    @Inject(MAT_DIALOG_DATA)
+    public data: any,
+    private dialogRef: MatDialogRef<OpenInAppComponent>
+  ) {
+    if (data) {
+      this.redirectUrl = data.redirectUrl;
+    }
+  }
+  ngOnInit(): void {}
+
+  OpenInApp() {
+    window.location.href = this.redirectUrl;
+  }
+  close(event) {
```

Comment:
you don't need this method at all, the "mat-dialog-close" already does this for you

Reply:
noted

Reply:
resolved.